RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/framework/SearchClusterContainer.java

```diff
@@ -44,22 +44,46 @@ public class SearchClusterContainer extends GenericContainer<SearchClusterContai
         "path.repo: \"/tmp/snapshots\""
     );
 
+    private static final Map<String, String> DISK_WATERMARK_COMMON = Map.of(
```

Comment:
Please review and restructure how the water mark settings are applied to a simplier pattern, if we added ES9 it would be hard to tell that we followed the correct behavior.